### ISSUE TITLE
Git based workflow for .atd files

### ISSUE THREAD

**Issue by wokalski** [dialohq/stork#12]
The git workflow should probably be first class (files workflow being a secondary citizen). I was thinking about something like:

```
stork gen 1:34fdsd3 2:fasfsdf 3:abc123 4 -o ./prefix/to/file <atdfilepath>
```

So the syntax is `<version_number>:<commit_hash>` and for the last one it's just `<version_number>`

Where the last one is for the current file.

**Comment by tsnobip** [comment]
yeah looks good, I'm pretty sure we will run into some issues when rebasing stuff in PRs but other than that I don't foresee any problems.

**Comment by wokalski** [comment]
I think it's not actually an issue. Because you don't care about the commit for the last one. You only care about the contents. And the previous ones should be on master already.

**Comment by tsnobip** [comment]
oh yeah that's right!

**Comment by wokalski** [comment]
You can use `git show <commit>:<atdfilepath>`

**Comment by tsnobip** [comment]
addressed by #22
